src/util/course_publish_helpers.py

```diff
@@ -22,5 +22,5 @@ def publish_waypoints(waypoints):
 
 def convert_waypoint_to_gps(waypoint_pose_pair: Tuple[Waypoint, SE3]) -> GPSWaypoint:
     waypoint, pose = waypoint_pose_pair
-    lat, lon, _ = pymap3d.enu2geodetic(pose.position[0], pose.position[1], pose.position[1], REF_LAT, REF_LON, 0.0)
+    lat, lon, _ = pymap3d.enu2geodetic(pose.position[0], pose.position[1], pose.position[2], REF_LAT, REF_LON, 0.0)
```

Comment:
Wait what is this? Is this a bug fix? I'm probably just missing something about this but not sure

Reply:
yea its a bug fix for the course publisher we were improperly setting the z coordinate to be the same as the y coordinate in the conversion. We probably never noticed it because with small coordinates the difference is really insignificant

Reply:
gotcha gotcha